spec.go

```diff
@@ -37,13 +37,21 @@ func (c *SpecController) Show(ctx *app.ShowSpecContext) error {
 	if len(elems) < 3 {
 		return fmt.Errorf("invalid package path %s", packagePath)
 	}
+	var branch string
+	parts := strings.Split(elems[len(elems)-1], "@")
+	design := parts[0]
+	if len(parts) > 1 {
+		branch = parts[1]
+		elems[len(elems)-1] = design
+		packagePath = strings.Join(elems, "/")
```

Comment:
Yes, it should work outside the `if`, though that computation is not really required unless there is a branch specified.